Gill_Kyle/details.html

```diff
@@ -0,0 +1,32 @@
+<%inherit file="app_base.htm" />
+
+<%block name="title">
+    <h1>${ page_title }</h1>
+</%block>
+
+<%block name="content">   
+    <div class="col-sm-8">
+        <div class="col-sm-2">
+        % for i in product_images:
+            <div class="thumbnail">
+                <img src="${i}" alt="alternate view of ${product.name}"/>
+            </div>
+        % endfor
+        </div>
+        <div class="col-sm-4">
+            <img class="thumbnail" id="large-img" src="${product_images[0]}" alt="${product.name}"/>
```

Comment:
product.image_url() might be preferred here

Reply:
Great point, no reason to have the method if I don't use it!